### PR TITLE
Adding Contribution.md file 

### LINKED ISSUE #58: [FEATURE] Add CONTRIBUTING.md file
### Description

- Add the contributing.md file explaining how to clone the repo and get started with contributing. 
- What are the requirements one should have before cloning the repo? 
- Mention how to propose a feature request? 
- How to create a pull request?  

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Santosh1176**
Signed-off-by: Santosh Kaluskar <dtshbl@gmail.com>

Added a CONTRIBUTION.md file, which explains:

- Add the contributing.md file explaining how to clone the repo and get started with contributing.
- What are the requirements one should have before cloning the repo?
- Mention how to propose a feature request?
- How to create a pull request?

## Fixes Issue
- `Closes #58` [issue. #58](https://github.com/kubesimplify/website/issues/58#issue-1253694737)

### CODE REVIEW

**Comment by kranurag7** [comment]
LGTM

File: CONTRIBUTION.md
Comment (AvineshTripathi) [inline CONTRIBUTION.md:107]:
```suggestion
git add  <file_name>
```

File: CONTRIBUTION.md
Comment (AvineshTripathi) [inline CONTRIBUTION.md:110]: I am not sure if we are using a DCO, should we?

cc @verma-kunal  @saiyam1814 @anubha-v-ardhan

File: CONTRIBUTION.md
Comment (AvineshTripathi) [inline CONTRIBUTION.md:17]: Should we give link to Docusaurus2.0 here as well?

Comment (Santosh1176) [reply CONTRIBUTION.md:17]: I think Docusaurus shouldn't be mentioned as there is an config.json file, IMHO, which is a key component of docusaurus framework. It mention of Docusaurus would help developers, know about it and work accordingly.

About DCO: I think it's becoming a SOP in open source projects to include DCO. This would help new contributors know about it and get used to. (Maintainers can be linient towards non-signed commits) Which would help them in future OSS contributions.

Above are my views. But, final call is your's.

**Review by NitishKumar06 — CHANGES_REQUESTED** [review]
This sounds good. A little suggestion:

**Comment by AvineshTripathi** [comment]
any particular reason why we want a $ before the git command @NitishKumar06

File: CONTRIBUTION.md
Comment (AvineshTripathi) [reply CONTRIBUTION.md:17]: Let the maintainers take the final decision, I am okay with both

**Comment by NitishKumar06** [comment]
@AvineshTripathi Well for people aiming for non-code contribution (and not actually coming from technical background), `$` will be good since it'll make them clear where to run these commands  else without using `$` , just make sure to include some text like _run the following commands on a terminal like git bash etc._

**Comment by Santosh1176** [comment]
@AvineshTripathi @kranurag7 plz recommend if any updates required to existing file, so as changes could be committed.

**Comment by kranurag7** [comment]
Great work @Santosh1176 and @AvineshTripathi I think we can go ahead and add the link to the main docusauras website there. Regarding the DCO we can request the contributors to force push their signature. We need to set up the DCO. I think it's good as of now. We will implement it very soon. Regarding code of conduct, we will be adding a separate file for that.

**Comment by Santosh1176** [comment]
Thanks, @kranurag7 that's great.  
Regarding Code-of-Conduct, shall I delete the section, or do you wish to amend that section in the future after the new code of conduct doc is published on the page with a new PR?

File: CONTRIBUTION.md
Comment (verma-kunal) [reply CONTRIBUTION.md:107]: This is absolutely important! Thanks for making this change @AvineshTripathi

File: CONTRIBUTION.md
Comment (verma-kunal) [reply CONTRIBUTION.md:110]: The guidelines looks nice, but I am not sure that we need these at this point of our project!
Posting this doubt in the channel

**Comment by AvineshTripathi** [comment]
LGTM sorry to keep you waiting


cc @verma-kunal

**Pull request merged**
